azerothcore/azerothcore-wotlk
# feat(Tools/Vmapextractor): Improved vmap detail level by extracting w…

…mo doodads (tables, chairs etc).

Port from TC.
Fixed #4782.

New Vmaps and Mmaps ARE REQUIRED.





## Changes Proposed:
-  
-  


## Issues Addressed:
- Closes #4782.



## Tests Performed:
-
-



## How to Test the Changes:

1. Enter Warsong Gulch.
2. Test one of these spots: https://user-images.githubusercontent.com/18535853/111451057-fe14b900-874b-11eb-8aed-62345a80b589.png https://user-images.githubusercontent.com/18535853/111451597-90b55800-874c-11eb-9b1d-2f2726040a46.png
3. Cast fear on self and observe results.
4. Additional check: Inside these "rooms" add temporary npc (e..g. `.npc add temp 1747`), move out the room, select the NPC and type `.debug los`, as I did here: https://user-images.githubusercontent.com/80540499/111452903-4499f180-8713-11eb-9432-c27af3223089.jpg. NPC should not be in LoS.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
